client/new-stack-editor/lib/views/credentiallistitem.coffee

```diff
@@ -0,0 +1,52 @@
+kd = require 'kd'
+JView = require 'app/jview'
+
+
+module.exports = class CredentialListItem extends kd.ListItemView
+
+  JView.mixin @prototype
+
+  constructor: (options = {}, data) ->
+
+    options.cssClass = kd.utils.curry 'credential', options.cssClass
+
+    super options, data
+
+    handle = (action) => =>
```

Comment:
This is gonna define a new handler for each credential list item, making it prototype method would be more efficient here, as it's gonna be defined and reused each time.